packages/cache/src/internal/cacheUtils.ts

```diff
@@ -117,8 +117,11 @@ export function getCacheFileName(compressionMethod: CompressionMethod): string {
 }
 
 export async function isGnuTarInstalled(): Promise<boolean> {
```

Comment:
Looks like this function is specifically for windows. Then rename it `isGnuTarInstalledOnWindows` just to ensure it is not used for linux by mistake.

Reply:
Good point

Reply:
Renamed to `getGnuTarPathOnWindows` inline with below comments